cookbook/console/command_in_controller.rst

```diff
@@ -0,0 +1,85 @@
+.. index::
+   single: Console; Use commands in your Controller
+
+How to make use of a Command in your application
+================================================
+
+The Console page of the Components section (:doc:`/components/console/introduction`) covers
+how to create a console command. This cookbook article covers how to use a console command
+directly in your application.
+
+Execute Command directly from Controller
+----------------------------------------
+
+Symfony allows you to directly execute a ``Command`` inside your Controller::
+
+    // src/AcmeBundle/Controller/GreetController.php
```

Comment:
Think I missed any notice about using Acme examples, so thanks for mentioning. Changing it right now is fine with me, please let me know when I can help :+1: